storybookjs/storybook
# Fix: vue2 events use bracket notation

Issue: #20753

## What I did

~~Fix casing for vue2 events to be camelcased by default when using the default storybook render fn.~~
Allow kebab case event names to work correctly in vue. 

## How to test

1. load vue2/vite sandbox. 
2. Add `this.$emit('also-clicked')` to line 51 of Button.vue
3. Remove the render function, line 8-12 from Button.stories.js
4. Add arg types `'also-clicked': { action: 'clicked' }` to argTypes
5. Error should be gone and should show event inside storybook actions.

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
